packages/typography/src/types.ts

```diff
@@ -1,3 +1,6 @@
 import { DarkModeProps } from '@leafygreen-ui/lib';
+import { BaseFontSize } from '@leafygreen-ui/tokens';
 
-export type CommonTypographyProps = DarkModeProps;
+export type CommonTypographyProps = DarkModeProps & {
+  baseFontSize?: BaseFontSize;
```

Comment:
Other components like `H1` and `H2` use `CommonTypographyProps` but they don't need to support `baseFontSize`

Reply:
I thought about that and have two conflicting opinions

1. I can see a world in which we do support "baseFontSize" in h1/h2 -- not that they would be 13/16px but that we change the font size of h1/h2 components based on whatever baseFontSize has been set. In this world, for now this is an optional prop and if you pass a value to the component it'll just be ignored so we're not introducing anything meaningful now and we can update later.
2. I can create a new type ResponsiveTypographyProps ? that inherits from Common and adds baseFontSize.

Which would you prefer to see here?

Reply:
I prefer option 2, i think it would be a little odd to see a component support a prop but not really do anything with it.

Reply:
Sounds good to me! Will make the change.

Reply:
Done!